hack/prebuild/local-dev/variables.tf

```diff
@@ -0,0 +1,61 @@
+variable "base_domain" {
+  type        = "string"
+  description = "The base DNS domain of the cluster."
+  default     = "tt.testing"
+}
+
+variable "libvirt_uri" {
+  type        = "string"
+  description = "libvirt connection URI"
+  default     = "qemu+tcp://192.168.122.1/system"
```

Comment:
Default connection for libvirt is `qemu:///system`. Why are we using a different one which needs first to be allowed in libvirt config?

Reply:
This is the default libvirt network, at least in my virbr0. Same value is used on the installer.
I'll switch it to `qemu:///system`.